### DISCUSSION TITLE
Does Laravel 10.x support PHP 8.3 officially yet?

### DISCUSSION THREAD

**Discussion by jameshulse** [laravel/framework#49576] [Q&A] [answered]
According to the Releases page [here](https://laravel.com/docs/10.x/releases#support-policy) it still lists 8.2. But I can see that Sail already supports 8.3, and there have been commits to add github actions to test against 8.3.

What is the official line?

**Comment by Jubeki** [comment, marked as answer]
Yes it does support PHP 8.3. I will send a PR to update the docs.

**Reply by jameshulse** [reply]
Fantastic. I couldn't make the decision to upgrade for our team until I saw it in the docs. Thanks.

**Reply by Jubeki** [reply]
It is also available in the docs now.

**Reply by jameshulse** [reply]
Fantastic, I can see that now. Will get the team upgraded to PHP 8.3 asap. Thanks for pushing that through.
